src/components/KeyAndTuningButton.tsx

```diff
@@ -22,6 +43,8 @@ export default function BasicButtons() {
         variant="outlined">
           {isIncomplete ? 'Select a root, mode, and tuning' : (isClicked ? 'Root, mode, and tuning confirmed' : 'Confirm root, mode, and tuning selection')}

```

Comment:
@stenzeladam for clarity's sake, you can put this in its own function and call it here. Nested ternary expressions can be difficult to understand.